Library/ViewModels/DashboardViewModel.swift

```diff
@@ -102,9 +102,6 @@ public protocol DashboardViewModelType {
 public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewModelOutputs,
   DashboardViewModelType {
 
-  //Project selected by user in the drawer or by push for the project message thread
-  private let selectedProjectProperty: SignalProducer<Param?, NoError>
```

Comment:
Sorry if I wasn't clear! I think all three `const`s can be local to the function (which includes the two mutable properties below this one).